src/lightning/__setup__.py

```diff
@@ -39,20 +39,32 @@ def _prepare_extras() -> Dict[str, Any]:
         for p in req_files
         if p.name not in ("docs.txt", "base.txt") and not p.parent.name.startswith("_")
     }
+
     # project specific extras groups
     extras["fabric-all"] = extras["fabric-strategies"] + extras["fabric-examples"]
     extras["fabric-dev"] = extras["fabric-all"] + extras["fabric-test"]
     extras["pytorch-all"] = extras["pytorch-extra"] + extras["pytorch-strategies"] + extras["pytorch-examples"]
     extras["pytorch-dev"] = extras["pytorch-all"] + extras["pytorch-test"]
-    extras["app-extra"] = extras["app-cloud"] + extras["app-ui"] + extras["app-components"]
+    extras["app-extra"] = extras["app-app"] + extras["app-cloud"] + extras["app-ui"] + extras["app-components"]
     extras["app-all"] = extras["app-extra"]
     extras["app-dev"] = extras["app-all"] + extras["app-test"]
+    extras["data-data"] += extras["app-app"]  # todo: consider cutting/leaning this dependency
     extras["data-all"] = extras["data-data"] + extras["data-cloud"] + extras["data-examples"]
     extras["data-dev"] = extras["data-all"] + extras["data-test"]
+    extras["store-store"] = extras["app-app"]  # todo: consider cutting/leaning this dependency
+
     # merge per-project extras of the same category, e.g. `app-test` + `fabric-test`
     for extra in list(extras):
         name = "-".join(extra.split("-")[1:])
         extras[name] = extras.get(name, []) + extras[extra]
+
+    # drop quasi base the req. file has the same name sub-package
+    for k in list(extras.keys()):
```

Comment:
```suggestion
    for extra in extras:
```

Reply:
not really, because in this case you can't edit the dst you are iterating